app/controllers/fe/concerns/answer_pages_controller_concern.rb

```diff
@@ -33,8 +33,8 @@ def update
     questions = @presenter.all_questions_for_page(params[:id])
     questions.set_filter(get_filter)
     questions.post(answer_params, @answer_sheet)
-
     questions.save
+    Fe::UpdateReferenceSheetVisibilityJob.perform_later(@answer_sheet, questions.questions.collect(&:id))
```

Comment:
It doesn't work well to pass objects into a background job (`@answer_sheet`). Instead, pass `@answer_sheet.id` and do a `find` in the job.